test/functional/page_objects/dashboard_page.ts

```diff
@@ -562,7 +562,7 @@ export class DashboardPageObject extends FtrService {
 
     await this.gotoDashboardLandingPage();
 
-    await this.listingTable.searchForItemWithName(dashboardName);
+    await this.listingTable.searchForItemWithName(dashboardName, { escape: false });
```

Comment:
For some reason this escapes `-`. This was a legacy implementation that was not validated but nonetheless preserved, in this case unnecessary.